app/src/NativeBase/Containers/ListContainer.tsx

```diff
@@ -261,12 +267,21 @@ const ListContainer = (props: {
   }, [params?.options?.events, params?.search, params?.type])
 
   return (
-    <Theme>
-      <SafeArea>
-        <TopOfApp
-          showSearchButton
-          onSearchButtonPress={() => navigate(screens.search[params.type], '')}
-        />
+    <>
+      <TopOfApp
+        showSearchButton
+        onSearchButtonPress={() => navigate(screens.search[params.type], '')}
+      />
+      <VStack
+        _dark={{ backgroundColor: 'bgDarkMode.100' }}
```

Comment:
I think you can remove these `_dark` and `_light` settings here as the default does the same?